pkg/web/api/v1/transaction.go

```diff
@@ -76,18 +76,18 @@ type Envelope struct {
 	EnvelopeID         string                   `json:"envelope_id,omitempty"`
 	Direction          string                   `json:"direction"`
 	Remote             string                   `json:"remote,omitempty"`
-	ReplyTo            ulid.ULID                `json:"reply_to,omitempty"`
+	ReplyTo            *ulid.ULID               `json:"reply_to"`
 	IsError            bool                     `json:"is_error"`
 	Error              *trisa.Error             `json:"error,omitempty"`
 	Identity           *ivms101.IdentityPayload `json:"identity,omitempty"`
 	Transaction        *generic.Transaction     `json:"transaction,omitempty"`
 	Pending            *generic.Pending         `json:"pending,omitempty"`
 	SentAt             *time.Time               `json:"sent_at"`
 	ReceivedAt         *time.Time               `json:"received_at,omitempty"`
-	Timestamp          time.Time                `json:"timestamp"`
+	Timestamp          time.Time                `json:"timestamp,omitempty"`
 	PublicKeySignature string                   `json:"public_key_signature,omitempty"`
 	TransferState      string                   `json:"transfer_state,omitempty"`
-	Original           []byte                   `json:"original,omitempty"`
+	SecureEnvelope     *SecureEnvelope          `json:"secure_envelope,omitempty"`
```

Comment:
Basically what I did was replace the `original` field with `secure_envelope` which contains all the nested metadata. Some of that is duplicated above, but we need the fields at the top level for people to send envelopes.